x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -126,28 +120,26 @@ export function ServiceInventory() {
   return (
     <>
       <SearchBar />
-      <EuiPage>
-        <EuiFlexGroup direction="column" gutterSize="s">
-          {displayMlCallout ? (
-            <EuiFlexItem>
-              <MLCallout onDismiss={() => setUserHasDismissedCallout(true)} />
-            </EuiFlexItem>
-          ) : null}
+      <EuiFlexGroup direction="column" gutterSize="s">
+        {displayMlCallout ? (
```

Comment:
You can use `&&` instead of `?` and remove the `: null` when it's false.
```suggestion
        {displayMlCallout && (
```